api/python/cellxgene_census/tests/experimental/pp/test_online.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 from scipy import sparse
 
-from cellxgene_census.experimental.pp._online import MeanVarianceAccumulator
+from cellxgene_census.experimental.pp._online import MeanAccumulator, MeanVarianceAccumulator
```

Comment:
It is totally *not* your bug, but could you add a simple / quick test of CountsAccumulator?

Reply:
Sure!